src/test/java/io/kontur/disasterninja/service/layers/providers/BivariateLayerProviderTest.java

```diff
@@ -65,14 +65,14 @@ private void init() {
         );
         List<BivariateLayerLegendQuery.Indicator> indicators = List.of(
                 new BivariateLayerLegendQuery.Indicator("Indicator",
-                        "area_km2", "Area", List.of(List.of("neutral"), List.of("neutral")),
+                        "area_km2", "Area", null, List.of(List.of("neutral"), List.of("neutral")),
```

Comment:
It's nice everything works with `null` without an NPE, but maybe we should use an empty string in place of a missing emoji? I don't know for sure... What rules do we follow in such cases?

Reply:
From FE standpoint, I don't see a problem with null. At least we clearly see this data is missing, and it's easy to process

Reply:
I've looked at the DB schema it seems to use `NULL` as a missing value as well. 👌